packages/experimental/Link/src/LinkTokens.ts

```diff
@@ -0,0 +1,66 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { LinkTokens } from './Link.types';
+
+export const defaultLinkTokens: TokenSettings<LinkTokens, Theme> = (t: Theme) =>
+  ({
+    color: t.colors.link,
+    alignSelf: 'flex-start',
+    variant: 'body1',
+    inline: {
+      textDecorationLine: 'underline',
+    },
+    disabled: {
+      color: t.colors.neutralForegroundDisabled,
+      textDecorationLine: 'none',
+      focused: {
+        borderColor: t.colors.focusBorder,
+      },
+    },
+    hovered: {
```

Comment:
I think linkHovered/linkPressed/link/link might all be old color names too? everything else looks like a new alias token though.

Reply:
I want to say it's likely the token you're looking for is brandForegroundLink/Hover/Pressed? But I'd check redlines